jobs/director/templates/director.yml.erb

```diff
@@ -253,6 +253,12 @@ elsif p('blobstore.provider') == 'gcs'
   if_p('blobstore.encryption_key') do |encryption_key|
      blobstore_options['encryption_key'] = encryption_key
   end
+elsif p('blobstore.provider') == 'azure-storage'
+    blobstore_options = {
```

Comment:
intention seem to be 2 spaces to big but as wrong as all the others in this file 🫨

Reply:
Done